source/renderer/app/stores/ada/AdaRedemptionStore.js

```diff
@@ -251,14 +248,17 @@ export default class AdaRedemptionStore extends Store {
       this._reset();
       this.actions.ada.adaRedemption.adaSuccessfullyRedeemed.trigger({
         walletId,
-        amount: new BigNumber(transaction.amount).toFormat(DECIMAL_PLACES_IN_ADA),
+        amount: transaction.amount.toFormat(DECIMAL_PLACES_IN_ADA),
```

Comment:
@MarcusHurney I had to rever this part as the flow-errors were popping out due to the usage of `AdaTransaction` flow-declaration few lines above this one. This `AdaTransaction` flow-declaration should have been only used for typing responses of the Api calls and not the stores as in the store we actually get a Transaction domain object!

Reply:
@nikolaglumac 
Ok, sorry about that. It does make sense. With regards to the acceptance tests, I was under the impression that many of them were failing because of endpoints not being set up yet. Like importing a wallet via JSON file and restarting the node after an update. I was essentially trying to focus on resolving flow errors and doing manual tests until we get to the final review on `feature/ddw-321-v1-api-integration` at which point all tests must pass before merging with develop. I think I'm out of the loop on whether or not we currently have any tests that we _expect_ to fail, because I thought we did and that was the reasoning behind waiting to fix them all at the same time.